src/renderer/bootstrap.tsx

```diff
@@ -89,18 +90,22 @@ export async function bootstrap(App: AppComponent) {
 
   SentryInit();
 
-  await ClusterStore.createInstance().loadInitialOnRenderer();
+  const cs = ClusterStore.createInstance();
+
+  await cs.loadInitialOnRenderer();
+
   HotbarStore.createInstance();
   ExtensionsStore.createInstance();
   FilesystemProvisionerStore.createInstance();
   ThemeStore.createInstance();
+  TerminalStore.createInstance();
```

Comment:
Maybe a comment indicating that the order is important here?

Reply:
Good idea.